src/commands/update-organization.ts

```diff
@@ -36,6 +38,17 @@ export class UpdateOrganizationCommand extends BaseCliCommand<IUpdateOrganizatio
     public async performCommand(command: IUpdateOrganizationCommandArgs): Promise<void> {
         const template = await TemplateRoot.create(command.templateFile);
         const state = await this.getState(command);
+        let govCloudProvider: S3StorageProvider;
+
+        /**
+         * These additional providers and such are all over the place, since I added them as I went along.
+         * Would be great if we could centralize the govcloud provider stuff.
+         */
+         const creds = await AwsUtil.GetGovCloudCredentials();
```

Comment:
this is one of those places where you will benefit from grouping all these different things in a single class on utils